docs/NewOperators.md

```diff
@@ -0,0 +1,18 @@
+## How to implement a new operator in Glow
+
+> Example PR: https://github.com/pytorch/glow/pull/2334
+
+### Implementation
+* Create a new glow high level IR node in `ClassGen/NodeGen.cpp`.
+* Implement the `verify()` method for the new node in `Graph/Nodes.cpp`.
+* Implement a node creation method in `Graph/Graph.cpp`.
+* Implement logic to load models that contains the operator in `Importer/Caffe2ModelLoader.cpp` or `Importer/ONNXModelLoader.cpp` depending on which type of model the operator comes from. Add the operator to `Importer/CommonOperatorLoader.h` instead if the loading logic can be shared between Caffe2 and ONNX.
+* Create a new glow low level IR instruction in `ClassGen/InstrGen.cpp`
+* Implement the interpreter implementation of the operator in `Interpreter/InterpreterNodes.cpp`
+
+### Testing
+* Test the operator implementation by writing unit tests in `unittests/OperatorTest.cpp`.
+* Test the operator importing logic by creating a new model in `tests/models/caffe2Models` or `tests/models/onnx2Models` directory then write unit tests in `unittests/Caffe2ImporterTest.cpp` or `tests/unittests/OnnxImporterTest.cpp` using that model.
```

Comment:
provide guidance on how to handle scenarios when there is a common implementation.
Like having `tests/models/caffe2Models` is enough for test coverage.